server/data-service/src/main/java/edu/illinois/ncsa/incore/service/data/controllers/DatasetController.java

```diff
@@ -207,7 +207,7 @@ public List<Dataset> getDatasets(@ApiParam(value = "DataType of IN-CORE datasets
             return datasets;
         }
         //get all datasets that the user can read
-        Set<String> userMembersSet = authorizer.getAllMembersUserHasReadAccessTo(username, spaceRepository.getAllSpaces());
+        Set<String> userMembersSet = authorizer.getAllMembersUserHasReadAccessTo(username, spaceRepository.getAllSpaces(),groups);
```

Comment:
Just curious, no need for "this.groups"?

Reply:
I just followed similar format to username, here it was username so I just went with it 😆